redisbeat/scheduler.py

```diff
@@ -77,10 +77,10 @@ def merge_inplace(self, tasks):
         for task, score in old_entries:
             if not task:
                 break
-            debug("ready to loads old entry: ", str(task))
+            debug("ready to load old_entries: %s", str(task))
```

Comment:
Hi @poteri can this change solve `TypeError: not all arguments converted during string formatting` error?

Reply:
It does indeed.